.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Link to any related issues, companion PRs, or blockers. eg, `Fixes #37`
+
+## Overview
+
+A quick summary of what this PR is meant to do.
+
+## Discussion
+
+More context that you think is important for reviewers or our future selves looking at this PR.
+
+## Testing
+
+How did you test this? If a reviewer pulled this branch down and wanted to kick the tires, how would they do so?
```

Comment:
```suggestion
How did you test this? If a reviewer wanted to test this, what steps should they take to do so?
```

Reply:
Thanks for the suggestions :) I pulled in the other suggestion but on this one I think the original better conveys my intent.